tests/virtual_machines_test.go

```diff
@@ -258,7 +258,7 @@ var _ = Describe("Virtual Machines", func() {
 				Expect(newMac2.String()).To(Equal(mac2.String()))
 			})
 		})
-		//2179
+		//2179 TODO until we split the range to 2 separate non overlapping - then this test is expected to fail.
```

Comment:
you can add skip with a reason and then it will skip them if you want
see test/utils.go
bottom please for the predicates that you can use in order to skip

Reply:
I plan to remove these tests in a future dedicated task, where I will also add tests relevant to the sharding case